packages/terra-time-input/tests/wdio/time-input-twelve-hour-mobile-spec.js

```diff
@@ -1,13 +1,18 @@
 /* global browser, Terra, before */
 const viewports = Terra.viewports('medium');
 
+const ignoredA11y = {
```

Comment:
Is there an issue logged regarding the inaccessible color contrast? If so, can we include the issue link in a commend so we know why we disabled this rule?

Reply:
Right here. I'll add the comment to this test. https://github.com/cerner/terra-core/issues/1670

Reply:
This comment seems to be mentioning accessibly in regards to incorrect labels, not color contrast. Do these go hand in hand?

Reply:
I'll update the issue. We should resolve all the accessibility issues in that github issue that couldn't be resolved in this commit.